spec/features/comment/author/destroy_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'Comment destroy' do
+  include_context 'current user signed in'
+
+  def create_comment
+    fill_form(:comment, comment_attributes)
+    click_button('Create Comment')
+  end
+
+  let(:article) { create(:article, author: current_user) }
+  let(:comment_attributes) { attributes_for(:comment) }
+
+  background { visit article_path(article) }
+
+  scenario 'Author delete own comment' do
+    create_comment
+    click_link('delete')
+
+    expect(page).to have_selector('div.alert.alert-notice')
```

Comment:
Its normal if you only check for human readable text like 'You comment was deleted' because it's bad if you use implementation details like class names - frontend devs can edit it, you can edit it, every time you will need to fix this test